modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase

```diff
@@ -19,11 +19,9 @@ definition {
 		if ("${testPortalInstance}" == "true") {
 			PortalInstances.tearDownCP();
 		}
-
 		else {
 			ObjectAdmin.deleteAllCustomObjectsViaAPI();
 		}
-
```

Comment:
It looks like you added the spaces in the previous commit and removed them here. Please remove them from the original commit next time using interactive rebase.